tool/README.md

```diff
@@ -71,93 +88,121 @@ From the git GUI tool or from github.com directly:
 
-### Upload the DevTools binary to CIPD
-Using the commit hash you want to release DevTools from (this should match the
-commit hash for the tag you just created) and the [update.sh](https://github.com/dart-lang/sdk/blob/master/third_party/devtools/update.sh)
-script, build and upload the DevTools binary to CIPD.
+   ```shell
+   tool/tag_version.sh;
+   ```
 
-```shell
-cd $LOCAL_DART_SDK
-git rebase-update
-third_party/devtools/update.sh <your commit hash like 8881a7caa9067471008a8e00750b161f53cdb843>
-```
+### Upload the DevTools binary to CIPD
+- `TARGET_COMMIT_HASH` should be the commit hash you want to release DevTools from
+   - this should match the
+commit hash for the tag you just created
+   - and the [update.sh](https://github.com/dart-lang/sdk/blob/master/third_party/devtools/update.sh)
+script,
+   ```shell
+   TARGET_COMMIT_HASH=<Commit hash for the version bump commit>
+   ```
+
+- build and upload the DevTools binary to CIPD.
+
+   ```shell
+   cd $LOCAL_DART_SDK && \
+   git rebase-update && \
+   third_party/devtools/update.sh $TARGET_COMMIT_HASH;
+   ```
 
 ### Update the DevTools hash in the Dart SDK
 
-Create new branch for your changes:
-```shell
-cd $LOCAL_DART_SDK
-git new-branch dt-release
-```
-
-Update the `devtools_rev` entry in the Dart SDK 
-[DEPS file](https://github.com/dart-lang/sdk/blob/master/DEPS)
-with the git commit hash you just built DevTools from (this is
-the id for the CIPD upload in the previous step). See this 
-[example CL](https://dart-review.googlesource.com/c/sdk/+/215520).
-
-Verify that running `dart devtools` launches the version of DevTools you just released. You'll
-need to build the dart sdk locally to do this.
-```shell
-cd $LOCAL_DART_SDK
-gclient sync -D
-./tools/build.py -mrelease -ax64 create_sdk
-out/ReleaseX64/dart-sdk/bin/dart devtools  # On OSX replace 'out' with 'xcodebuild'
-```
-
-If the version of DevTools you just published to CIPD loads properly, push up the SDK CL for review.
-```shell
-git add .
-git commit -m "Bump DevTools DEP to 2.16.0"
-git cl upload -s
-```
+- Create new branch for your changes:
+   ```shell
+   cd $LOCAL_DART_SDK && \
+   git new-branch dt-release;
+   ```
+
+- Update the `devtools_rev` entry in the Dart SDK [DEPS file](https://github.com/dart-lang/sdk/blob/master/DEPS)
+   - set the `devtools_rev` entry to the `TARGET_COMMIT_HASH`.
+   - See this [example CL](https://dart-review.googlesource.com/c/sdk/+/215520) for reference.
+
+
+- Build the dart sdk locally
+
+   ```shell
+   cd $LOCAL_DART_SDK && \
+   gclient sync -D && \
+   ./tools/build.py -mrelease -ax64 create_sdk;
+   ```
+
+- Verify that running `dart devtools` launches the version of DevTools you just released. 
+   - For non-OSX
+      ```shell
+      out/ReleaseX64/dart-sdk/bin/dart devtools
+      ```
+   - for OSX
+      ```shell
+      xcodebuild/ReleaseX64/dart-sdk/bin/dart devtools
+      ```
+
+- If the version of DevTools you just published to CIPD loads properly
```

Comment:
make a note that you may need to clear your browser history or open in an incognito window to avoid issues with Chrome aggressively caching